src/style.css

```diff
@@ -0,0 +1,166 @@
+/* #region reset */
+:root {
+  --color-primary: #060b35;
+  --color-secondary: #616070;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-primary);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+h3 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+/* #endregion */
+
+/* #region stars */
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  min-width: 16px;
+  min-height: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url("images/star-active.svg");
+}
+
+/* #endregion */
+
+/* #region card */
+
+.container {
+  max-width: 200px;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product {
+  max-width: 160px;
+  height: 134px;
+  display: flex;
+  margin: 0 auto;
+}
+
+.card__img {
+  width: 100%;
+}
+
+.card__name {
+  margin-top: 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.card__code {
+  color: var(--color-secondary);
+  display: block;
+}
+
+.code {
+  margin-top: 4px;
+}
+
+.reviews {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin: 16px 0 24px;
+}
+
+.price {
```

Comment:
no need to create bem block for `code, reviews and price`

Reply:
<img width="693" alt="image" src="https://user-images.githubusercontent.com/102411130/230842664-da40aa00-cf23-4979-8b4f-85960f2f241e.png">